SecurityPkg/Include/Protocol/MuTcg2Protocol.h

```diff
@@ -38,11 +38,35 @@ EFI_STATUS
   IN TPML_DIGEST_VALUES   *DigestList,
   IN EFI_TCG2_EVENT       *Event
   );
+
+/**
+  The EFI_MU_TCG2_PROTOCOL MuEventInit function call provides callers with
+  an interface for initializing events before they are logged.
+
+  Supported Event Types:
+  EV_NO_ACTION
+  EV_EFI_HCRTM_EVENT
+
+  @param[in]      This             Indicates the calling context

```

Comment:
There is not a `This` parameter on the actual function signature. There should be for consistency across the protocol API.